frameworks/kafka/docs/ssl-auth.md

```diff
@@ -1,184 +0,0 @@
----
-layout: layout.pug
-navigationTitle: 
-excerpt:
-title: SSL Auth
-menuWeight: 23
-
----
-
-## How to connect an authenticated client
-
-### Create a DC/OS CA signed certificate (Note: This can be replaced with some enterprise CLI calls)
```

Comment:
These instructions here seem to be missing from the updated docs. Have they been moved elsewhere?

Reply:
I condensed it to the section `SSL Auth/Authenticating a Client`.

I didn't want to lay out all the openssl commands and junk, but still tell them it requires the DC/OS CA.

Reply:
OK. Sounds good. Where do we document this for ourselves?

Reply:
Well... it's documented in the tests...